### PR TITLE
Update FCM client

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: runtime/config.go [hunk 1]
```diff
@@ -73,7 +73,8 @@ type Config struct {
 	CourierAuthToken string `help:"the authentication token used for requests to Courier"`
 	LibratoUsername  string `help:"the username that will be used to authenticate to Librato"`
 	LibratoToken     string `help:"the token that will be used to authenticate to Librato"`
-	FCMKey           string `help:"the FCM API key used to notify Android relayers to sync"`
+
+	AndroidFCMServiceAccountFile string `help:"the FCM Service Account Credentials JSON File used to notify Android relayers to sync"`
```
Comment (rowanseymour) [inline runtime/config.go:77]: Doesn't it need project ID Too?

Comment (norkans7) [reply runtime/config.go:77]: Not needed

File: core/msgio/courier.go [hunk 2]
```diff
@@ -152,7 +152,7 @@ func NewCourierMsg(oa *models.OrgAssets, m *models.Msg, u *models.ContactURN, ch
 		}
 	}
 
-	if m.Contact != nil {
+	if m.Contact != nil && m.Contact.LastSeenOn() != nil {
```
Comment (rowanseymour) [inline core/msgio/courier.go:155]: Why?

**Review by rowanseymour — APPROVED** [review]

File: go.mod [hunk 3]
```diff
@@ -34,18 +35,34 @@ require (
 )
 
 require (
+	cloud.google.com/go v0.114.0 // indirect
```
Comment (rowanseymour) [inline go.mod:38]: so many new deps :(

Comment (norkans7) [reply go.mod:38]: Yeah, I think we find a way around that later

**Pull request merged**